lib/app/models/testing_instance.rb

```diff
@@ -187,6 +187,11 @@ def conformance_supported?(resource, methods = [])
         end
       end
 
+      def operation_supported?(op_name)
+        operation = server_capabilities.capabilities.rest&.first&.operation&.select { |x| x.name == op_name }
```

Comment:
This is only testing the server-level operations. I think you also need to test the resource-level operations (at `rest.resource.operations`) since the IG supports `export` for Patient and Group. I feel like this should use `any?` rather than `first` and `select`?

Also, could you use a descriptive variable name rather than `x`?

Reply:
This method is redundant. Removed from the latest fix.

Reply:
The version in `ServerCapabilities` has the same problem. It's only checking server-level operations, but (correct me if I'm wrong) `Patient` and `Group` exports are resource-level operations, so it needs to check operations at the resource level (i.e. check `capabilities.rest.resource.operation` rather than only `capabilities.rest.operation`).

Reply:
According to the IG capability statement (http://hl7.org/fhir/uv/bulkdata/STU1/CapabilityStatement-bulk-data.json.html), those operations are declared in rest ```rest``` level. Your comment also make sense. I would like to keep align with current IG and I will contact IG author to figure out if IG need to be updated.